catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/source/MiroTransformableData.scala

```diff
@@ -1,37 +1,34 @@
-  @JsonProperty("image_creator") creator: Option[List[String]],
-  @JsonProperty("image_image_desc") description: Option[String],
-  @JsonProperty("image_image_desc_academic") academicDescription: Option[
-    String],
-  @JsonProperty("image_secondary_creator") secondaryCreator: Option[
-    List[String]],
-  @JsonProperty("image_artwork_date") artworkDate: Option[String],
-  @JsonProperty("image_cleared") cleared: Option[String],
-  @JsonProperty("image_copyright_cleared") copyrightCleared: Option[String],
-  @JsonProperty("image_keywords") keywords: Option[List[String]],
-  @JsonProperty("image_keywords_unauth") keywordsUnauth: Option[List[String]],
-  @JsonProperty("image_phys_format") physFormat: Option[String],
-  @JsonProperty("image_lc_genre") lcGenre: Option[String],
-  @JsonProperty("image_tech_file_size") techFileSize: Option[List[String]],
-  @JsonProperty("image_use_restrictions") useRestrictions: Option[String],
-  @JsonProperty("image_supp_lettering") suppLettering: Option[String],
-  @JsonProperty("image_innopac_id") innopacID: Option[String],
-  @JsonProperty("image_credit_line") creditLine: Option[String],
-  @JsonProperty("image_source_code") sourceCode: Option[String],
-  @JsonProperty("image_library_ref_department") libraryRefDepartment: Option[
-    List[String]],
-  @JsonProperty("image_library_ref_id") libraryRefId: Option[List[String]],
-  @JsonProperty("image_award") award: Option[List[String]],
-  @JsonProperty("image_award_date") awardDate: Option[List[String]]
+  @JsonKey("image_title") title: Option[String],
+  @JsonKey("image_creator") creator: Option[List[String]],
+  @JsonKey("image_image_desc") description: Option[String],
+  @JsonKey("image_image_desc_academic") academicDescription: Option[String],
+  @JsonKey("image_secondary_creator") secondaryCreator: Option[List[String]],
+  @JsonKey("image_artwork_date") artworkDate: Option[String],
+  @JsonKey("image_cleared") cleared: Option[String],
+  @JsonKey("image_copyright_cleared") copyrightCleared: Option[String],
+  @JsonKey("image_keywords") keywords: Option[List[String]],
+  @JsonKey("image_keywords_unauth") keywordsUnauth: Option[List[String]],
+  @JsonKey("image_phys_format") physFormat: Option[String],
+  @JsonKey("image_lc_genre") lcGenre: Option[String],
+  @JsonKey("image_tech_file_size") techFileSize: Option[List[String]],
+  @JsonKey("image_use_restrictions") useRestrictions: Option[String],
+  @JsonKey("image_supp_lettering") suppLettering: Option[String],
+  @JsonKey("image_innopac_id") innopacID: Option[String],
+  @JsonKey("image_credit_line") creditLine: Option[String],
+  @JsonKey("image_source_code") sourceCode: Option[String],
+  @JsonKey("image_library_ref_department") libraryRefDepartment: List[Option[String]] = Nil,
+  @JsonKey("image_library_ref_id") libraryRefId: List[Option[String]] = Nil,
+  @JsonKey("image_award") award: List[Option[String]] = Nil,
+  @JsonKey("image_award_date") awardDate: List[Option[String]] = Nil
```

Comment:
These have become `List[Option[String]]` instead of `Option[List[String]]`. Check comment on `MiroTransformableTransformer` as to why